ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/OracleRunner.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml
+package http.perf
+
+import dbutils.ConnectionPool
+import http.dbbackend.{DbStartupMode, JdbcConfig}
+import testing.oracle.OracleAround
+
+import scala.util.{Success, Try}
+
+private[this] final class OracleRunner {
+
+  private val defaultUser = "ORACLE_USER"
+  private val retainData = sys.env.get("RETAIN_DATA").exists(_ equalsIgnoreCase "true")
+  private val useDefaultUser = sys.env.get("USE_DEFAULT_USER").exists(_ equalsIgnoreCase "true")
+  type St = OracleAround.RichOracleUser
+
+  def start(): Try[St] = Try {
+    if (useDefaultUser) OracleAround.createOrReuseUser(defaultUser)
+    else OracleAround.createNewUniqueRandomUser()
+  }
+
+  def jdbcConfig(user: St): JdbcConfig = {
+    import DbStartupMode._
+    val startupMode: DbStartupMode = if (retainData) CreateIfNeededAndStart else CreateAndStart
+    JdbcConfig(
+      dbutils.JdbcConfig(
```

Comment:
The naming seems a bit confusing. You can tell them apart from the package name of course, but still likely to be causing confusion.